### PR TITLE
M3 redux account

### PR DESCRIPTION
**Pull request by Jskobos**
## Description

Replace the Account context in UpdateContactInformationPanel with a Redux container. This allows us to (among other things) automatically update the VAT banner state when the form is submitted.

## Note to Reviewers

~This also includes the VAT commits, so marking as don't merge~

### CODE REVIEW

File: src/features/Billing/BillingPanels/UpdateContactInformationPanel/UpdateContactInformationPanel.tsx [hunk 1]
```diff
@@ -99,26 +100,26 @@ class UpdateContactInformationPanel extends React.Component<
...
   };
 
   renderLoading = () => null;
 
   renderErrors = (e: Linode.ApiFieldError[]) => null;
 
   renderForm = (account: Linode.Account) => {
-    const { classes } = this.props;
-    const { fields, submissionErrors, success } = this.state;
+    const { classes, accountError } = this.props;
```
Comment (martinmckenna) [inline src/features/Billing/BillingPanels/UpdateContactInformationPanel/UpdateContactInformationPanel.tsx]: This form isn't being filled with the initial values. Most likely to do something with RenderGuard??

File: src/features/Billing/BillingPanels/UpdateContactInformationPanel/UpdateContactInformationPanel.tsx [hunk 2]
```diff
@@ -135,7 +136,7 @@ class UpdateContactInformationPanel extends React.Component<
         'tax_id',
         'zip'
       ],
-      submissionErrors
+      accountError.update
     );
 
     const generalError = errorMap.none;
```
Comment (martinmckenna) [inline src/features/Billing/BillingPanels/UpdateContactInformationPanel/UpdateContactInformationPanel.tsx]: This error also isn't rendering anymore

File: src/features/Billing/BillingPanels/UpdateContactInformationPanel/UpdateContactInformationPanel.tsx (hunk 1 above)
Comment (Jskobos) [reply src/features/Billing/BillingPanels/UpdateContactInformationPanel/UpdateContactInformationPanel.tsx]: Fixed that too, but I think this was just that the order of arguments to withAccount changed elsewhere so this whole thing broke.

File: src/store/account/account.reducer.ts [hunk 3]
```diff
@@ -36,7 +37,42 @@ const reducer: Reducer<State> = (state: State = defaultState, action) => {
...
+      ...state,
+      loading: false,
+      error: { ...state.error, read: payload }
+    };
+  }
+
+  if (isType(action, updateAccountActions.started)) {
+    return {
+      ...state,
+      loading: true
```
Comment (martinmckenna) [inline src/store/account/account.reducer.ts]: we should clear error state here

**Pull request merged**